front/src/common/redux/teachers/teachers.actions.js

```diff
@@ -0,0 +1,9 @@
+import { SHOW_TEACHER_REGISTER_MODAL, HIDE_TEACHER_REGISTER_MODAL, REGISTER_TEACHER, SHOW_LOADER } from './types';
```

Comment:
wrap to different lines in accordance with the general style of the code

Reply:
Done